Backends/Wayland/package.lisp

```diff
@@ -0,0 +1,39 @@
+(in-package #:common-lisp-user)
+
+(uiop:define-package #:clim-wayland
```

Comment:
please don't use uiop:define-package - in mcclim codebase uiop is rather discouraged (although there are a few uses in asd definitions and in the listener)

Reply:
done